cmd/subctl/root.go

```diff
@@ -53,6 +55,8 @@ func init() {
 	runtime.Must(submarineropv1a1.AddToScheme(scheme.Scheme))
 
 	rest.SetDefaultWarningHandler(suppressWarnings{})
+
+	log.SetLogger(logr.New(log.NullLogSink{}))
```

Comment:
Why not use `log.SetLogger(logr.Discard())` ?

Reply:
Same effect but `Discard` is more readable.

Reply:
Well `Discard` passes a nil `LogSink` which causes a nil pointer panic:

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x38 pc=0x21e16b0]

goroutine 1 [running]:
sigs.k8s.io/controller-runtime/pkg/log.(*loggerPromise).Fulfill(0xc000463c70, {0x0?, 0x0?})
	sigs.k8s.io/controller-runtime@v0.15.0/pkg/log/deleg.go:67 +0x30
sigs.k8s.io/controller-runtime/pkg/log.(*loggerPromise).Fulfill(0xc000463c20, {0x0?, 0x0?})
	sigs.k8s.io/controller-runtime@v0.15.0/pkg/log/deleg.go:83 +0x17d
sigs.k8s.io/controller-runtime/pkg/log.(*delegatingLogSink).Fulfill(...)
	sigs.k8s.io/controller-runtime@v0.15.0/pkg/log/deleg.go:192
sigs.k8s.io/controller-runtime/pkg/log.SetLogger({{0x0?, 0x0?}, 0x4c58fa0?})
	sigs.k8s.io/controller-runtime@v0.15.0/pkg/log/log.go:50 +0x4b
```

So I went back to using `NullLogSink`.